app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -171,7 +171,7 @@
             android:enabled="false"
             android:text="@string/sign_up"
             app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="@+id/cbTC"
+            app:layout_constraintStart_toStartOf="parent"
```

Comment:
This change is already taken care of  #617

Reply:
> This change is already accomodated in PR #617
I am not sure if we have to check the PRs also for some changes in the existing PRs.
Your suggestions are welcome.

Thanks